### PR TITLE
RLM-322 Change openstack_hosts role to rpc one

### PR DESCRIPTION
**Pull request by Frank-ZhangXin**
Link the rpc openstack_hosts role which is removed period mark from
templates/openstack-host-hostfile-setup.sh.j2

### CODE REVIEW

File: ansible-role-requirements.yml [hunk 1]
```diff
@@ -10,3 +10,7 @@
   scm: git
   src: https://github.com/ceph/ansible-ceph-osd.git
   version: 434a13f4d3ee3f60c410052d2670d5dfea034bf8
+- name: openstack_hosts
```
Comment (antonym) [inline ansible-role-requirements.yml]: Let's add a comment here detailing why we're forking the repo and include the JIRA number as well so we can reference this change easily in the future.

Comment (antonym) [reply ansible-role-requirements.yml]: Other than that, it looks good, points to the https://github.com/rcbops/openstack-ansible-openstack_hosts/tree/leapupgrade-version which is the change on top of the newton code.

**Comment by antonym** [comment]
recheck_all

File: ansible-role-requirements.yml (hunk 1 above)
Comment (Frank-ZhangXin) [reply ansible-role-requirements.yml]: Yeah, gonna add comment to explain the separated role.

**Comment by antonym** [comment]
recheck_cit_xenial_swift_deploy

**Comment by Frank-ZhangXin** [comment]
@antonym comment is added to the role requirement.

File: ansible-role-requirements.yml [hunk 2]
```diff
@@ -10,3 +10,10 @@
   scm: git
   src: https://github.com/ceph/ansible-ceph-osd.git
   version: 434a13f4d3ee3f60c410052d2670d5dfea034bf8
+# Created for JIRA RLM-322
+# Frank: this is a forked ansible role for removing
+# hardcoded period mark from hostname template.
+- name: openstack_hosts
+  scm: git
+  src: https://github.com/rcbops/openstack-ansible-openstack_hosts
+  version: ce249ec4386d013d76b617f009a93a408f76619e
```
Comment (antonym) [inline ansible-role-requirements.yml]: Merged in PR on openstack-ansible-openstack_hosts, updated stable/newton version is now: 235ec6f0fc5570e7c657b243bcf4208cc2c7739c

Comment (Frank-ZhangXin) [reply ansible-role-requirements.yml]: cool, update is on the way.

**Pull request merged**